bundles/org.openhab.binding.wemo/src/main/java/org/openhab/binding/wemo/internal/WemoHandlerFactory.java

```diff
@@ -96,6 +98,22 @@ protected ThingHandler createHandler(Thing thing) {
                 logger.debug("Creating a WemoDimmerHandler for thing '{}' with UDN '{}'", thing.getUID(),
                         thing.getConfiguration().get("udn"));
                 return new WemoDimmerHandler(thing, upnpIOService, wemoHttpcaller);
+            } else if (thingTypeUID.equals(WemoBindingConstants.THING_TYPE_CROCKPOT)) {
+                logger.debug("Creating a WemoCockpotHandler for thing '{}' with UDN '{}'", thing.getUID(),
+                        thing.getConfiguration().get("udn"));
```

Comment:
I'm not sure if getting a configuration option is a good thing here. If the user updates the configuration it's possible to set a different value for udn and thus what would have been printed here is not valid anymore. So it might be better to log this in the initialize method of the handler.

Reply:
The UDN is discovered via UPnP and set in registry, so not meant to be changed.